src/_clean.sh

```diff
@@ -66,5 +66,5 @@ function clean_sweepUploadsCache() {
   info "core-backend: Cleaning uploads/exports caches"
   # Leaving the cleanup fairly light, this should help a ton without getting aggressive
   debug "`compose_client exec -T -w /usr/src/plextrac-api/uploads plextracapi \
-    find . -maxdepth 1 -type f -regextype posix-extended -regex '^.*\.(json|xml|ptrac|csv|nessus)$' -delete`"
```

Comment:
Need to understand better if there was a reason that `-regextype posix-extended` was chosen...

Reply:
> Need to understand better if there was a reason that `-regextype posix-extended` was chosen...

https://www.gnu.org/software/findutils/manual/html_node/find_html/posix_002dextended-regular-expression-syntax.html#posix_002dextended-regular-expression-syntax

Reply:
@BadgerOps Thank you!

Reply:
No problem. Now, why @jesseops added that is still a mystery that doc strings or a semi snarky GitHub comment could possibly fix.